src/Orneholm.CognitiveWorkbench.Web/Views/Vision/CustomVision.cshtml

```diff
@@ -0,0 +1,239 @@
+@using Orneholm.CognitiveWorkbench.Web.Extensions
+@using Microsoft.Azure.CognitiveServices.Vision.CustomVision.Prediction.Models
+@model CustomVisionViewModel
+<main role="main" class="pb-3">
+    <div class="container">
+        <h2>Vision - Custom Vision</h2>
+
+        <p>
+            Process classification of images or object detection based on your own tags.
+        </p>
+
+        <form method="post">
+            <div class="card mb-3">
+                <div class="card-header">
+                    <ul class="nav nav-tabs card-header-tabs">
+                        <li class="nav-item">
+                            <a class="nav-link active" data-toggle="list" href="#cwb-form-customvision-image">Image</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" data-toggle="list" href="#cwb-form-customvision-settings">Settings</a>
+                        </li>
+                    </ul>
+                </div>
+                <div class="card-body">
+                    <div class="tab-content">
+                        <div class="tab-pane fade show active" id="cwb-form-customvision-image">
+                            <p class="mb-3">
+                                <ul class="list-unstyled text-small">
+                                    <li>
+                                        The image must be presented in JPEG, PNG, GIF, or BMP
+                                    </li>
+                                    <li>
+                                        The file size of the image must be less than 4 megabytes (MB)
+                                    </li>
+                                    <li>
+                                        The dimensions of the image must be greater than 256 pixels for both height and width (otherwise, they will be upscaled)
+                                    </li>
+                                    <li>
+                                        See service documentation for limits and quotas on <a href="https://docs.microsoft.com/en-us/azure/cognitive-services/Custom-Vision-Service/limits-and-quotas">docs.microsoft.com</a>
+                                    </li>
+                                </ul>
+                            </p>
+
+                            <p class="mb-3">
+                                <strong>Note:</strong> You need to provide your own keys for Azure Cognitive Services - Computer Vision under the tab <em>Settings</em>.
```

Comment:
COpy mistake here. Change to custom vision.

Reply:
You're right, a bad copy paste as the links were also wrong. Fixed